src/app/angular/cdk-tree/cdk-tree.component.ts

```diff
@@ -11,14 +11,17 @@ import { CdkTreeService } from './cdk-tree.service';
 })
 export class CdkTreeComponent {
   static selector = 'appCdkTree';
-  private readonly treeData: ExampleFlatNode[] = this.cdkTreeService.getTreeData();
-  treeControl = new FlatTreeControl<ExampleFlatNode>(
+  private readonly treeData: ExampleFlatNode[] = [];
```

Comment:
I don't think we need to initialize this (and doing so might cover some failures in the future, if we are trying to access this before initializing it with actual data). I think it would be better to leave it uninitialized and initialize it in the constructor:

```suggestion
  private readonly treeData: ExampleFlatNode[];
```

Reply:
Fixed